packages/plus/src/components/EstimateCost/locales/en.ts

```diff
@@ -0,0 +1,22 @@
+export default {
```

Comment:
should we add localazy into the CI for this one ?

Reply:
@philibea Im not sure there is a need for it as we are not planning to provide any other language than english